libenclave/src/lib.rs

```diff
@@ -12,16 +12,16 @@
 #![feature(linkage,lang_items,unwind_attributes,asm,const_fn,collections,unicode,alloc,oom,heap_api)]
 #![no_std]
 
-#[macro_use] extern crate collections;
-extern crate rustc_unicode;
+extern crate collections;
+extern crate std_unicode;
 extern crate alloc as rustc_alloc;
 extern crate sgx_isa;
 #[cfg(not(test))] pub extern crate core_io as io;
 
 extern crate spin;
 extern crate rlibc;
 extern crate alloc_buddy_simple;
-#[macro_use] extern crate bitflags;
+extern crate bitflags;
```

Comment:
Done, I also removed `rlibc` and `std_unicode`. They didn't seem to be used anywhere else.